cli/tests/pcluster/validators/test_ebs.py

```diff
@@ -123,3 +125,29 @@ def test_ebs_size_iops_validators(section_dict, expected_message):
 def test_ebs_volume_type_size_validator(section_dict, expected_message):
     actual_failures = SharedStorageSchema().load({"MountDir": "/my/mount/point", "EBS": section_dict}).validate()
     assert_failure_messages(actual_failures, expected_message)
+
+
+@pytest.mark.parametrize(
+    "volume_dict, expected_message",
```

Comment:
instead of defining the volume_dict you can directly use kms_key_id and encrypted as test parameters.
To test the validator you don't need the entire volume structure, you just need the two parameters to be tested.

Reply:
So you can avoid to define the Size.

Reply:
Done